patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,17 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Used as a last resort for determining potion effect sort order in the gui.
++     * This can be used to preserve a potion effect's sort order if it has a liquid color that is subject to change.
```

Comment:
You are inconsistent with "potion effect" and "PotionEffect". Please stick to one, and if you use `PotionEffect`, enclose it in `{@code}` like `{@code PotionEffect}`.